nflow-engine/src/main/resources/scripts/db/mysql.legacy.create.ddl.sql

```diff
@@ -108,6 +109,7 @@ create table if not exists nflow_archive_workflow (
   retries int not null default 0,
   created timestamp not null,
   modified timestamp not null,
+  started timestamp,
```

Comment:
nflow_archive_workflow doesn't have `null`, nflow_workflow has `null`. Shouldn't the both fields be identical?

Reply:
Yes, fixed